source/etherpad.rst

```diff
@@ -0,0 +1,194 @@
+.. highlight:: console
+
+.. sidebar:: About
+
+  This guide has been tested with Uberspace 7.1.1 and Etherpad Lite 1.6.3
+
+#############
+Etherpad Lite
+#############
+
+Etherpad Lite is a real-time collaborative writing tool. It is based on NodeJs and comes with lot of possible plugins.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * Node.js_ and its package manager npm_ 
+  * MySQL_ 
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We're using Node.js_ in the stable version 8:
+
+::
+
+ [etherpad@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '8'
+ [etherpad@stardust ~]$ 
+
+You'll need your MySQL_ credentials. Get them with ``my_print_defaults``:
+
+::
+
+ [etherpad@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=etherpad
+ --password=MySuperSecretPassword
+ [etherpad@stardust ~]$ 
+
+Your domain needs to be setup:
+
+::
+
+ [etherpad@stardust ~]$ uberspace web domain list
+ etherpad.uber.space
+ [etherpad@stardust ~]$ 
+
+Installation
+============
+
+First get the Etherpad Lite source code from Github_:
+
+.. code-block:: console
+
+  [etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
```

Comment:
What's the output of that? What should the user expect? I suggest changing to:

```
[etherpad@stardust ~]$ git clone https://github.com/ether/etherpad-lite ~/etherpad
Cloning into '/home/etherpad/etherpad'...
remote: Counting objects: 29789, done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 29789 (delta 9), reused 16 (delta 7), pack-reused 29768
Receiving objects: 100% (29789/29789), 19.25 MiB | 20.58 MiB/s, done.
Resolving deltas: 100% (21251/21251), done.
[etherpad@stardust ~]$ 
```

This way it's totally clear to everybody if it worked or not.